src/lib/core/utils/axis-range-margin.ts

```diff
@@ -6,40 +6,42 @@ export function axisRangeMargin(
   valueType?: string | undefined
 ): NumberOrDateRange | undefined {
   // compute truncated axis with 5 % area from the range of min and max
-  if (valueType != null && valueType === 'date') {
-    // find date diff (days) between range.min and range.max, take 5 % of range, and round up!
-    const dateRangeDiff = Math.round(
-      DateMath.diff(
+  if (valueType != null) {
```

Comment:
```
if (valueType != null && axisRange != null) {
```
Checking for non-null `axisRange` would avoid the need for all the `?`s in the `axisRange?.min` and `axisRange?.max` below, I think?

Reply:
yep I think so, but will check and test it!